docs/getting-started.md

```diff
@@ -115,6 +115,7 @@ Then configure webpack to convert `app.scss` into `bundle.css`. For that you nee
 
 ```js
 module.exports = [{
+  mode: "production",
```

Comment:
The mode shouldn't be set in webpack.config.js; it should be set in package.json at line 386 under the section ### Step 5: Build Assets for Production. The command to compile the SCSS and JS files is npm run build which utilizes the build script at line 386.

Reply:
`-p` already does that.

Reply:
> `-p` already does that.

The production mode should still be used in the webpack build script because it also enables `FlagDependencyUsagePlugin`, `FlagIncludedChunksPlugin`, `ModuleConcatenationPlugin`, `NoEmitOnErrorsPlugin`, `OccurrenceOrderPlugin`, and `SideEffectsFlagPlugin`. The -p option only enables `optimization.minimize`. **Using the mode is also recommended by the webpack as it outputs the warning** below if the mode isn't set using webpack --mode production.
<pre>
<samp>
WARNING in configuration
The 'mode' option has not been set, webpack will fallback to 'production' for this value. Set 'mode' option to 'development' or 'production' to enable defaults for each environment.
You can also set it to 'none' to disable any default behavior. Learn more: https://webpack.js.org/configuration/mode/
</samp>
</pre>

Reply:
Can you link to the docs that say it is recommended? I'm not getting this warning

Reply:
> Can you link to the docs that say it is recommended? I'm not getting this warning

If you type in <kbd>npx webpack</kbd> into the terminal using webpack 4, the warning will appear. <a href="https://medium.com/webpack/webpack-4-mode-and-optimization-5423a6bc597a">Medium.com</a> says "_So webpack 4 added a mode option. And it’s required. (Actually it’s not required but it will warn you if you omit it.)_." This is also confirmed by a <a href="https://stackoverflow.com/questions/49242756/how-to-set-mode-to-development-or-production-in-the-config-file">StackExchange post</a>.